core/rawdb/accessors_metadata_ext.go

```diff
@@ -0,0 +1,112 @@
+// (c) 2025, Ava Labs, Inc.
+package rawdb
+
+import (
+	"time"
+
+	"github.com/ava-labs/libevm/common"
+	"github.com/ava-labs/libevm/ethdb"
+	"github.com/ava-labs/libevm/rlp"
+)
+
+// writeTimeMarker writes a marker of the current time in the db at [key]
```

Comment:
nit:
```suggestion
// writeTimeMarker writes a marker of the current time in the db at `key`
```
and in other places